src/app.py

```diff
@@ -72,37 +77,71 @@ def test_endpoint():
     },
 ]
 
-
-#@app.route("/fortune", methods=["GET"])
-
+# Helper function
 def get_all_fortunes():
     # connect to the database
     dynamodb = boto3.resource('dynamodb', region_name='us-west-2')
 
     # Access database table
     table = dynamodb.Table('FortuneCookie')
 
-    fortune = "Your high-minded principles spell success."
-
-    #response_obj = {"status": "success"}
     try:
         response_obj = table.scan()
     except ClientError as e:
-        print("Failed...")
         print(e.response['Error']['Message'])
     else:
         items = response_obj['Items']
         return items
         # testing prints
         #print("GetItem succeeded: ")
         #print(json.dumps(item, indent=4, cls=DecimalEncoder))
-    #return jsonify(response_obj) this is for Dustin
 
+def delete_fortune(id):
+    # connect to the database
```

Comment:
you can delete this since the next line is basically the same thing.

Reply:
Updated. :)